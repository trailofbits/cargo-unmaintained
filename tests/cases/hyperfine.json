[
  {
    "name": "bitvec",
    "outdated_deps": [
      {
        "name": "radium",
        "req": "^0.7",
        "version_latest": null,
        "version_used": "0.7.0"
      },
      {
        "name": "wyz",
        "req": "^0.5",
        "version_latest": null,
        "version_used": "0.5.1"
      },
      {
        "name": "rand",
        "req": "^0.8",
        "version_latest": null,
        "version_used": "0.8.5"
      }
    ],
    "repo_status": {
      "Age": null
    },
    "version": "1.0.1"
  },
  {
    "name": "borsh-derive-internal",
    "outdated_deps": [],
    "repo_status": "Unassociated",
    "version": "0.10.3"
  },
  {
    "name": "borsh-schema-derive-internal",
    "outdated_deps": [],
    "repo_status": "Unassociated",
    "version": "0.10.3"
  },
  {
    "name": "cloudabi",
    "outdated_deps": [
      {
        "name": "bitflags",
        "req": "^1.0",
        "version_latest": null,
        "version_used": "1.3.2"
      }
    ],
    "repo_status": {
      "Age": null
    },
    "version": "0.0.3"
  },
  {
    "name": "fuchsia-cprng",
    "outdated_deps": [],
    "repo_status": "Nonexistent",
    "version": "0.1.1"
  },
  {
    "name": "rand_os",
    "outdated_deps": [],
    "repo_status": "Unassociated",
    "version": "0.1.3"
  },
  {
    "name": "rdrand",
    "outdated_deps": [
      {
        "name": "rand_core",
        "req": "^0.3",
        "version_latest": null,
        "version_used": "0.3.1"
      }
    ],
    "repo_status": {
      "Age": null
    },
    "version": "0.4.0"
  },
  {
    "name": "statistical",
    "outdated_deps": [
      {
        "name": "num",
        "req": "^0.2",
        "version_latest": null,
        "version_used": "0.2.1"
      },
      {
        "name": "rand",
        "req": "^0.6",
        "version_latest": null,
        "version_used": "0.6.5"
      }
    ],
    "repo_status": {
      "Age": null
    },
    "version": "1.0.0"
  },
  {
    "name": "utf8parse",
    "outdated_deps": [],
    "repo_status": "Unassociated",
    "version": "0.2.1"
  }
]
