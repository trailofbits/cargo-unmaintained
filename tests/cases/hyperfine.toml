url = "https://github.com/sharkdp/hyperfine"
rev = "v1.18.0"
