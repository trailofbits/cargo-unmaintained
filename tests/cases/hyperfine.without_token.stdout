fuchsia-cprng (no repository)
atty (https://github.com/softprops/atty updated [..] days ago)
    hermit-abi (requirement: ^0.1.6, version used: 0.1.19, latest: [..])
cloudabi (https://github.com/nuxinl/cloudabi updated [..] days ago)
    bitflags (requirement: ^1.0, version used: 1.3.2, latest: [..])
statistical (https://github.com/JeffBelgum/statistical updated [..] days ago)
    num (requirement: ^0.2, version used: 0.2.1, latest: [..])
    rand (requirement: ^0.6, version used: 0.6.5, latest: [..])
